src/Mod/SoldierNamePool.cpp

```diff
@@ -129,6 +143,36 @@ std::string SoldierNamePool::genName(SoldierGender *gender, int femaleFrequency)
 	return name.str();
 }
 
+/**
+ * Returns a new random callsign from the
+ * lists of names contained within.
+ * @param gender Returned gender of the callsign.
+ * @return The soldier's callsign.
+ */
+std::string SoldierNamePool::genCallsign(const SoldierGender gender) const
+{
+	std::ostringstream callsign;
```

Comment:
What is point for `std::ostringstream` instead of `std::string`?

Reply:
No point, changing it.